r/R/dplyr-functions.R

```diff
@@ -414,6 +414,25 @@ nse_funcs$pmax <- function(..., na.rm = FALSE) {
   )
 }
 
+nse_funcs$str_pad <- function(string, width, side = c("left", "right", "both"), pad = " ") {
+  
+  side <- match.arg(side)
```

Comment:
Let's do some more argument validation to make better error messages if users pass unexpected inputs:
```suggestion
  assert_that(is_integerish(width))
  side <- match.arg(side)
  assert_that(is.string(pad))
```

Reply:
I think this is important because `stringr::str_pad()` can take vector `width` and `pad` and recycle them, but that will not work here.